### DISCUSSION TITLE
tailwindcss usages visualizer

### DISCUSSION THREAD

**Discussion by michael-land** [tailwindlabs/tailwindcss#6066] [Help] [answered]
is there any existing tool similar to [windicss-analysis](https://github.com/windicss/windicss-analysis)

![image](https://user-images.githubusercontent.com/33362998/141521646-22f6d85a-84de-453d-bad0-c0ffc09e0c84.png)

**Comment by adamwathan** [comment, marked as answer]
I often use https://cssstats.com/ for this sort of thing, but we've talked for a while about making something more Tailwind specific, even as recently as yesterday! So nothing exists right now but I think it's something we'll explore 👍🏻
